chukwuemekachm/utility_manager_web
# Prof: User search data

#### What does this PR achieve?
- users can now search for data in the settings page
- moves DateHelper method to dateHelpers
- updates sagas and reducers to make searching possible

#### Any background context?
- Made some changes to the backend that would make this complete

#### Manual Testing Steps
- Navigate to the settings page
- Search for units, parameters or appliance category
- You should be able to filter
- Also, you should be able to view ohms as a symbol correctly

#### Screenshots:
#### Relevant Trello Tickets
- [User should search data](https://trello.com/c/WWiopAYV/107-users-should-be-able-to-search-for-data-in-the-settings-page)
